test/game_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
+require "game"
+require "response"
+
+class GameTest < Minitest::Test
+  def test_can_create_instances
+    game = Game.new
+
+    assert_instance_of Game, game
+  end
+
+  def test_contains_default_instance_variables
+    game = Game.new
+
+    assert_equal 25, game.goal
+    assert_equal 0, game.guessed_count
+  end
+
+  def test_can_check_a_guess
+    game = Game.new
+    game.guessed_num = 24
+    assert_equal -1, game.guess_checker
```

Comment:
Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the `-` if it should be a subtraction.